<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Base</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="loginDialogTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Log in</string>
      </attribute>
      <widget class="QPushButton" name="loginButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>250</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Log in</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="loginUsernameEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>amanjiang</string>
       </property>
       <property name="maxLength">
        <number>39</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="loginPasswordEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>190</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>pswcom%@11</string>
       </property>
       <property name="maxLength">
        <number>72</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Sign up</string>
      </attribute>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Reenter password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="signupPasswordEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>72</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QPushButton" name="signupButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>250</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Sign up</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="signupUsernameEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>39</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="signupPasswordEdit2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>190</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>72</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>loginUsernameEdit</tabstop>
  <tabstop>loginPasswordEdit</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>loginDialogTab</tabstop>
  <tabstop>signupUsernameEdit</tabstop>
  <tabstop>signupPasswordEdit</tabstop>
  <tabstop>signupPasswordEdit2</tabstop>
  <tabstop>signupButton</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources/BaseClient.qrc"/>
 </resources>
 <connections/>
</ui>
